streaming/src/test/scala/org/apache/spark/streaming/ReceivedBlockHandlerSuite.scala

```diff
@@ -251,9 +377,20 @@ class ReceivedBlockHandlerSuite
     (blockIds, storeResults)
   }
 
+  /** Store block using a handler */
+  private def storeBlock(
```

Comment:
Yeah, I know that. I had meant use that for single block. Could have done 
```
def storeSingleBlock(params): (.., ...) = {
  storeBlocks(params).head
}
```
Would have further deduped code. But this is a nit. This is fine.